en/mkdocs.yml

```diff
@@ -689,6 +686,9 @@ nav:
           - 'Configuring the Tenant Loading Policy': administer/configuring-the-tenant-loading-policy.md
           - 'Adding New Tenants': administer/adding-new-tenants.md
       - 'LDAP vs JDBC': administer/ldap-vs-jdbc.md
+      - 'Working with the Registry':
+        - 'Working with the Registry': administer/working-with-the-registry.md
```

Comment:
can we change this title or parent one? having the same name might not give a clear idea.

Reply:
This pattern is used for navigations in left side bar

Reply:
Yes, that's what I'm referring to. this navigation item and its parent have the same name.